code/ai/aicode.cpp

```diff
@@ -542,6 +542,21 @@ void garbage_collect_path_points()
 
 }
 
+void reset_ai_path_points()
+{
+	memset(Path_points, 0, sizeof(Path_points)); // optional: zero out for safety
+	Ppfp = Path_points;
+
+	// Also clear any AI path state
+	for (int i = 0; i < MAX_SHIPS; ++i) {
+		if (Ai_info[i].path_length > 0) {
```

Comment:
This might be paranoia, but it might be safer to omit the check out and just reset it regardless.